src/arch/mod.rs

```diff
@@ -149,9 +160,11 @@ impl Type {
             Error => Err("cannot take `alignof` <type error>"),
         }
     }
+    /// Get the pointer type
```

Comment:
This is not really an accurate description. This returns an integer type which is large enough to represent the pointer; this is an IR-level type and has no relation to any C type.